scripts/claude_mcp.sh

```
@@ -0,0 +1,1733 @@
+
+    # Wait for all installations to complete
+    echo -e "${BLUE}  ⏳ Waiting for $batch_name installations to complete...${NC}"
+
+    for server_name in "${!batch_ref[@]}"; do
+        local pid="${PARALLEL_PIDS[$server_name]}"
+        local result_file="/tmp/install_${server_name}.result"
+        local log_file="/tmp/install_${server_name}.log"
+
+        # Wait for this installation
+        if wait "$pid"; then
+            local result="$(cat "$result_file" 2>/dev/null || echo "UNKNOWN")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)